src/main/resources/ValidationSequenceMessages.properties

```diff
@@ -2,4 +2,5 @@ SedimentCheck=Value \"{0}\" is not in the required set of 'Sedimentation coeffic
 MarkerCheck=Value \"{0}\" is not in the required set of 'Markers', Permitted values are {1} 
 MandatoryFieldsCheck=The following mandatory field(s) are missing {0} - All headers are capitalised.
 HeadersValuesCheck=The following header(s) are missing values: {0} Please provide value(s) for the listed header(s). Please check that all headers have values for ALL sequences before re-submitting
-HeadersSupportedCheck=The following header(s) are not supported: {0} Please remove them before re-submitting
\ No newline at end of file
+HeadersSupportedCheck=The following header(s) are not supported: {0} Please remove them before re-submitting
+SampleSupportedCheck=The given Organism: \"{0}\" is not a sample. If is added as sample please check if the sample is associated with the passed user.
```

Comment:
- I suggest we remove the ```SampleSupportedCheck``` message because there might be a very large number of these info messages.